mathics/builtin/image.py

```diff
@@ -1238,15 +1239,19 @@ def make_boxes(self, form):
                 scaled_height = int(scale * height)
                 pixels = skimage.transform.resize(pixels, (scaled_height, scaled_width), order=0)
 
-            stream = BytesIO()
-            skimage.io.imsave(stream, pixels, 'pil', format_str='png')
-            stream.seek(0)
-            contents = stream.read()
-            stream.close()
+            with warnings.catch_warnings():
+                warnings.simplefilter("ignore")
```

Comment:
Not really. During the conversion of floating point to byte components (which are necessary for the png encoding), PIL outputs a warning that precision might be lost. Since this code only handles the display for the frontend, it doesn't matter.